priv/templates/smartos/Makefile

```diff
@@ -127,29 +133,29 @@ packing_list_files: $(BUILD_STAGE_DIR) templates
 	   echo "@comment Packing /opt/local/etc files" >> +CONTENTS && \
 	   echo "@cwd /opt/local" >> +CONTENTS && \
 	   echo "@owner root" >> +CONTENTS && \
-	   echo "@group {{package_install_user}}" >> +CONTENTS && \
+	   echo "@group riak" >> +CONTENTS && \
```

Comment:
Yep, I'll get right on it. Heh, and sorry for my denseness, I guess you're right it clearly wasn't intentional :).